text/runtime-printing.md

```diff
@@ -37,7 +37,7 @@ use sp_runtime::print;
 
 ## Tests
 
-Next we see a reference to the tests module. This pallet, as with most recipes pallets, has tests written in a separate file called
+Following, we see a reference to the tests module. This pallet, as with most recipes pallets, has tests written in a separate file called
```

Comment:
I think "next" is a more appropriate word to use here.

Reply:
Yes, I just found that that word's use is a little repetitive in here.